activestorage/test/controllers/direct_uploads_controller_test.rb

```diff
@@ -108,8 +108,10 @@ class ActiveStorage::DiskDirectUploadsControllerTest < ActionDispatch::Integrati
   test "creating new direct upload" do
     checksum = Digest::MD5.base64digest("Hello")
 
+    ActiveStorage::DirectUploadsController.skip_forgery_protection
     post rails_direct_uploads_url, params: { blob: {
       filename: "hello.txt", byte_size: 6, checksum: checksum, content_type: "text/plain" } }
+    ActiveStorage::DirectUploadsController.protect_from_forgery with: :exception
```

Comment:
@georgeclaghorn 

I fully agree with you.  I removed the changes and add commit to post authenticity token.